test/expect/TestJit.test_lstm_fusion_cpu.expect

```diff
@@ -6,31 +6,25 @@ graph(%0 : Float(3, 10)
       %5 : Float(80)
       %6 : Float(80)) {
   %7 : Float(10!, 80!) = aten::t(%3)
-  %8 : Float(3, 80) = aten::addmm[beta={1}, alpha={1}](%5, %0, %7)
-  %9 : Float(20!, 80!) = aten::t(%4)
-  %10 : Float(3, 80) = aten::addmm[beta={1}, alpha={1}](%6, %1, %9)
-  %11 : Float(3!, 20), %12 : Float(3!, 20), %13 : Float(3!, 20), %14 : Float(3!, 20) = aten::chunk[chunks=4, dim=1](%8)
-  %15 : Float(3!, 20), %16 : Float(3!, 20), %17 : Float(3!, 20), %18 : Float(3!, 20) = aten::chunk[chunks=4, dim=1](%10)
-  %19 : Float(3, 20), %20 : Float(3, 20) = prim::FusionGroup_0[device=-1](%2, %14, %18, %13, %17, %12, %16, %11, %15)
-  return (%19, %20);
+  %8 : Float(3!, 80) = aten::expand[size=[3, 80], implicit=0](%5)
+  %9 : Dynamic = aten::addmm[alpha={1}, beta={1}](%8, %0, %7)
+  %10 : Float(20!, 80!) = aten::t(%4)
+  %11 : Float(3!, 80) = aten::expand[size=[3, 80], implicit=0](%6)
+  %12 : Dynamic = aten::addmm[alpha={1}, beta={1}](%11, %1, %10)
+  %13 : Float(3, 80) = aten::add[alpha={1}](%9, %12)
```

Comment:
This add should have been fused into the FusionGroup, but `addmm` doesn't have a correct output shape here, preventing it from happening. This is likely a perf. regression. Whatever code introduces the addmm should propagate the shape correctly.